src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -68,11 +68,24 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
   @Input()
   public id: string = `sky-checkbox-${++nextId}`;
 
+  /**
+   * Indicates whether the checkbox is disabled.
+   */
+  public get disabled() {
```

Comment:
Getters should come immediately after the setters.

Reply:
Noted. Fixed in last commit